index.bs

```diff
@@ -5154,10 +5087,10 @@ for {{DOMException}}, a description, and legacy code values.
 Note: If an error name is not listed here, please file a bug as indicated at the top of this specification and it will be addressed shortly. Thanks!
 
 Note: Don't confuse the "{{SyntaxError!!exception}}" {{DOMException}} defined here
-with ECMAScript's {{ECMAScript/SyntaxError}}.
+with ECMAScript's {{SyntaxError}}.
 "{{SyntaxError!!exception}}" {{DOMException}} is used to report parsing errors in web APIs,
 for example when parsing selectors,
-while the ECMAScript {{ECMAScript/SyntaxError}} is reserved for the ECMAScript parser.
+while the ECMAScript {{SyntaxError}} is reserved for the ECMAScript parser.
 To help disambiguate this further,
 always favor the "{{SyntaxError!!exception}}" {{DOMException}} notation
 over just using {{SyntaxError!!exception}} to refer to the {{DOMException}}. [[DOM]]
```

Comment:
The new `{{SyntaxError}}` references are incorrect: it goes to the unrelated `"SyntaxError"` `DOMException` in Web IDL rather than ES. In fact, this paragraph explains exactly how they are different.